app/views/catalog/_show_top_fields.html.erb

```diff
@@ -0,0 +1,16 @@
+<% document = @document %>
+<% doc_presenter = show_presenter(document) %>
+<dl class="row">
+  <% document_show_fields(document).each do |field_name, field| -%>
+    <% if render_top_field? document, field, field_name %>
+      <dt class="col-3 text-right blacklight-<%= field_name.parameterize %>-label"><%= render_document_show_field_label(document, field: field_name).html_safe %></dt>
+      <% if doc_presenter.configuration.show_fields.fetch(field_name)[:helper_method] %>
```

Comment:
@cdmo I think we need a check for multi-valued fields for fields like additional creators like here: https://github.com/psu-libraries/psulib_blacklight/blob/master/app/views/catalog/_show_default.html.erb#L7?

Reply:
I refactored this a bunch. Can you [take a look when you have a chance](https://github.com/psu-libraries/psulib_blacklight/pull/336/commits/0c362e5c7ae16ee2b0bad896c91c3f2072e604a6)?

Reply:
these changes looks great 👌 way cleaner 👍